versions.json

```diff
@@ -165,6 +165,9 @@
             "jsVersion": "24.0.0-alpha4",
             "npmName": "@vaadin/message-list"
         },
+        "mpr-v8": {
```

Comment:
do we also need that `mpr-v7`? seems we have both in the spring-bom
https://github.com/vaadin/platform/blob/master/scripts/generator/templates/template-vaadin-spring-bom.xml#L555...L564

Reply:
oh.. now i noticed your Note..